snuba/consumers/consumer_builder.py

```diff
@@ -217,10 +260,15 @@ def __build_consumer(
                 commit_retry_policy=self.__commit_retry_policy,
             )
 
-        return StreamProcessor(consumer, self.raw_topic, strategy_factory, IMMEDIATE)
+        # Can we assume that the raw topic would be mapped
+        # to a physical topic in the sliced context?
```

Comment:
do you know the answer to this question? seems like it should be "yes", and if so we can remove the comment